### PR TITLE
Unit test for reduct_query function for no filters, one filter, and two filters

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Unit test for `reduct_query` function (happy path that was not covered)

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Unit tests

### CODE REVIEW

File: tests/unit/app/endpoints/test_ols.py [hunk 1]
```diff
@@ -139,9 +139,21 @@ def test_validate_question_on_validation_error(validate_question_mock, load_conf
         ols.validate_question(conversation_id, llm_request)
 
 
+@patch("ols.utils.config.query_redactor")
+def test_query_filter_on_correct_reduct(mock_redact_query, load_config):
```
Comment (bparees) [inline tests/unit/app/endpoints/test_ols.py:143]:
```suggestion
def test_query_filter_on_correct_redact(mock_redact_query, load_config):
```

File: tests/unit/app/endpoints/test_ols.py [hunk 2]
```diff
@@ -139,9 +139,21 @@ def test_validate_question_on_validation_error(validate_question_mock, load_conf
...
+    """Test the function to redact query."""
+    conversation_id = suid.get_suid()
+    query = "Tell me about Kubernetes"
+    llm_request = LLMRequest(query=query, conversation_id=conversation_id)
+    mock_redact_query.redact_query.return_value = "Mocked query"
+    result = ols.redact_query(conversation_id, llm_request)
+    assert result is not None
+    assert result.query == "Mocked query"
```
Comment (bparees) [inline tests/unit/app/endpoints/test_ols.py:151]: this is another case of "i don't really see the point of this test".  It seems to be just checking the return type of ols.redact_query.

Can't we have a test that:
1) actually verifies that the redaction happens
2) *and* confirms the return type

together?  This test alone hardly seems worth having+maintaining.  If ols.redact_query starts returning a different object, i'd expect us to quickly notice that when everything else starts failing.

Comment (tisnik) [reply tests/unit/app/endpoints/test_ols.py:151]: ok that sounds a bit like a plan :)

Comment (tisnik) [reply tests/unit/app/endpoints/test_ols.py]: now it might be better @bparees , WDYT?

**Pull request merged**